cmd/oras/internal/display/status/discard.go

```diff
@@ -139,3 +139,28 @@ func (DiscardHandler) OnIndexPacked(ocispec.Descriptor) error {
 func (DiscardHandler) OnIndexPushed(string) error {
 	return nil
 }
+
+// OnBlobPushSkipped implements BlobPushHandler.
```

Comment:
nit:

```suggestion
// OnBlobExists implements BlobPushHandler.
```

Same for other similar places.

Reply:
changed